src/bin/rav1e.rs

```diff
@@ -231,6 +236,7 @@ fn do_encode<T: Pixel, D: Decoder>(
     &mut buffer,
     &mut buf_pos,
     y4m_enc.as_mut(),
+    parse_cli().unwrap().metrics_enabled,
```

Comment:
We should pull this out and store it in a variable up above the `while` loop (i.e. line 229). Right here, this still calls `parse_cli` on every iteration of the loop.